docs/store/selectors.md

```diff
@@ -97,13 +97,23 @@ export interface AppState {
   feature: FeatureState;
 }
 
-export const selectFeature = createFeatureSelector<FeatureState>('feature');
+export const selectFeature = createFeatureSelector<AppState, FeatureState>(
+  'feature'
+);
 export const selectFeatureCount = createSelector(
   selectFeature,
   (state: FeatureState) => state.counter
 );
 ```
 
+Meaning that the following would not compile because `foo` is not a feature slice of `AppState`.
+
```

Comment:
Change to `The following selector below would not ...`

Reply:
No problem! Thanks for the reviews!